scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/BaseOxfordNanoporeProcessor.java

```diff
@@ -180,5 +238,22 @@ public synchronized NotificationDto process(File runDirectory, TimeZone tz) thro
     }
   }
 
+  /**
+   * Tests whether file at path is older than Jan 1 2017.
+   *
+   * @param path path leading to file to test
+   * @return true if file is older than Jan 1 2017 OR IOException occurs, false otherwise
+   */
+  protected boolean olderThanCutoff(Path path) {
+    try {
+      return Files.readAttributes(path, BasicFileAttributes.class)
+              .creationTime()
+              .compareTo(CUTOFF_DATE)
+          <= 0;
+    } catch (IOException e) {
+      return true;
```

Comment:
This method should throw the IOException rather than pretending it means something that it doesn't. It's used within another method that already throws IOException anyway

Reply:
Done